clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/HubConnectionBuilder.java

```diff
@@ -30,10 +31,16 @@ public HubConnectionBuilder configureLogging(Logger logger) {
         return this;
     }
 
+    // For testing purposes only
+    HubConnectionBuilder configureHttpClient(HttpClient client) {
```

Comment:
Why not just use the ctor you added?

Reply:
Because I'm trying to steer the tests towards https://github.com/aspnet/SignalR/blob/release/2.2/test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionTests.Helpers.cs#L10